src/components/ciPipeline/ciPipeline.css

```diff
@@ -160,7 +160,7 @@
     margin-right: 10px;
 }
 
-.info__container.info__container--linked-ci {
+.dc__info__container.info__container--linked-ci {
```

Comment:
can we wrap it in some parent container class?

Reply:
removed this and added mb-16 manually